src/library/Collections.nim

```diff
@@ -1403,6 +1411,15 @@ proc defineLibrary*() =
         },
         returns     = {Range},
         example     = """
+        ; range of :integers
+        array range 0 5     ; [0 1 2 3 4 5]
+        @0..5               ; [0 1 2 3 4 5]
+        ..........
+        ; range of :chars
+        @'a'..'e'           ; [a b c d e]
+        ..........
+        ; range with steps
+        range.step: 2 1 5   ; [1 3 5]
```

Comment:
Can we make this a bit more generic?

I mean... if I read the example, I would think that range always needs `array` (or `@`) which is precisely the point: ranges are "lazy" arrays, that don't become one until/unless needed.

So, if you don't mind, I'd suggest:
- remove the `@`s
- the "results" could be more verbal and not a value (e.g. `; all integers between 0 and 5, inclusive")
- you could add an extra example *with* `@` only to highlight that *this*, yes, is an array (aka "evaluated range" 😉 )

Reply:
You're completely right, I'll fix it very soon. :wink: